lib/nylas/v2/model/attribute_definition.rb

```diff
@@ -0,0 +1,20 @@
+module Nylas
+  module V2
+    module Model
+      class AttributeDefinition
+        extend Forwardable
```

Comment:
What does `Forwardable` do?

Reply:
Forwardable is a Ruby module for delegating methods from Instance A to Attribute B. It's a useful tool when relying on composition for organization of responsibility.